src/Host/Views/Consent/Index.cshtml

```diff
@@ -8,9 +8,19 @@
                 <div class="client-logo"><img src="@Model.ClientLogoUrl"></div>
```

Comment:
Note.   I have urls inserted here and this isnt working due to CORS errors.    Not sure what you have been inserting.